saleor/graphql/webhook/subscription_types.py

```diff
@@ -878,6 +881,35 @@ class Meta:
         interfaces = (Event,)
 
 
+class PermissionGroupBase(AbstractType):
+    permission_group = graphene.Field(
+        "saleor.graphql.account.types.Group",
+        description="The permission group the event relates to."
+        + ADDED_IN_35
```

Comment:
- We shouldn't include the `ADDED` and `PREVIEW_FEATURE` in `Base` type. It is inherited by all child events. It means that each event will get `ADDED_IN_35` (which could be not true for future added events). 
- Also `ADDED_IN_35` -> `ADDED_IN_36`
- Also `ADDED_IN_XX` and `PREVIEW_FEATURE` should be included in `Meta` section for each event like `PermissionGroupCreated` (with proper description of the event).

Reply:
So that should be applied to all old events, maybe I will create separate PR after this one, to update the description for all events?